docs_samples/SimpleApps/module1.py

```diff
@@ -0,0 +1,29 @@
+# module1.py
+import logging
+import os
+
+from opencensus.trace import config_integration
+from opencensus.trace.samplers import AlwaysOnSampler
+from opencensus.trace.tracer import Tracer
+from opencensus.ext.azure.log_exporter import AzureLogHandler
+from module2 import function_1
+
+config_integration.trace_integrations(['logging'])
+logging.basicConfig(format='%(asctime)s spanId=%(spanId)s %(message)s')
+tracer = Tracer(sampler=AlwaysOnSampler())
+
+logger = logging.getLogger(__name__)
+
+appKey = os.getenv('APP_INSIGHTS_KEY')
+
+logger.addHandler(AzureLogHandler(
+    connection_string=appKey)
+)
+
+logger.warning('Before the span')
```

Comment:
What is the purpose of these apps that are not being referenced by an instructions file? A lot of these samples are already in the main Opencensus azure monitor exporter [examples](https://github.com/census-instrumentation/opencensus-python/tree/master/contrib/opencensus-ext-azure/examples) folder, so there is not much point in them being here.

Reply:
I thought it ideal to have everything in one project and easily executable via all the setup scripts/templates.